cypress/utils/table.js

```diff
@@ -33,9 +33,24 @@ function tableIsSortedBy(columnTitle) {
     .should('have.class', 'pf-c-table__sort pf-m-selected');
 }
 
+function checkEmptyState(title, body, checkIcon = false) {
+  cy.get(TABLE)
+    .find('[ouiaid=empty-state]')
+    .should('have.length', 1)
+    .within(() => {
+      cy.get('.pf-c-empty-state__icon').should(
+        'have.length',
+        checkIcon ? 1 : 0
+      );
+      cy.get(`h5${TITLE}`).should('have.text', title);
+      cy.get('.pf-c-empty-state__body').should('have.text', body);
```

Comment:
Maybe just checking the title is enough. If anytime the body text changes, we do not need to update anything. But it is also fine to test for it.

Reply:
this make sense :)